libevm/VM.cpp

```diff
@@ -543,6 +544,7 @@ void VM::interpretCases()
 
 		CASE_BEGIN(BALANCE)
 		{
+			m_runGas += toUint64(m_schedule->balanceGas);
```

Comment:
It previously was charged ~20 gas. Are you sure it will not be now 20 + balanceGas?